src/agent/coverage/examples/block_coverage.rs

```diff
@@ -55,7 +55,7 @@ fn main() -> Result<()> {
     };
 
     let mut cmd = Command::new(&opt.cmd[0]);
-    cmd.args(&opt.cmd[1..]);
+    cmd.stdin(Stdio::null()).args(&opt.cmd[1..]);
```

Comment:
This is the fix for the issue i was having when reading inputs from the terminal.
It seems like at least one of the process we start with Command is reading from stdin.
So i added stdin(Stdio::null()) to all the command we launch